backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/cgen/CBridgeGen.kt

```diff
@@ -756,6 +756,14 @@ private fun KotlinStubs.mapType(type: IrType, retained: Boolean, variadic: Boole
 private fun IrType.isTypeOfNullLiteral(): Boolean = this is IrSimpleType && hasQuestionMark
         && classifier.isClassWithFqName(KotlinBuiltIns.FQ_NAMES.nothing)
 
+private fun IrType.isVector(): Boolean {
+    if (this is IrSimpleType) {
+        val fqName = FqName("kotlinx.cinterop.NativeVector").toUnsafe()  // FIXME
+        return classifier.isClassWithFqName(fqName)
```

Comment:
Nullability (`this.hasQuestionMark`) must also be checked.

Reply:
Done.